docs/topics/mta-web-reviewing-an-analysis-report.adoc

```diff
@@ -4,16 +4,16 @@
 
 :_content-type: PROCEDURE
 [id="mta-web-reviewing-an-analysis-report_{context}"]
-= Reviewing an analysis report
+= Reviewing analysis details
```

Comment:
It is my understanding that the analysis details is not the same as the old analysis report.

@jortel, @dymurray - In older versions, was there a link to open the windup report from the application details drawer?  Is that what the older docs are referencing?

If that is the case, the analysis report is now just either downloading the static report or the Issues and Dependencies from Details tab of the application details drawer.